R/backend-.R

```diff
@@ -280,11 +280,16 @@ base_scalar <- sql_translator(
   str_trim = sql_str_trim,
   str_c = sql_paste(""),
   str_sub = sql_str_sub("SUBSTR"),
+  # https://docs.getdbt.com/sql-reference/like is typically case sensitive
   str_like = function(string, pattern, ignore_case = TRUE) {
```

Comment:
I wonder if `ignore_case` should default to `FALSE`? I filed an issue in stringr to make sure to fix it there too: https://github.com/tidyverse/stringr/issues/543.

Reply:
Yes, I do see that but for now I have left this as is in this pr because it seems that would first need to change in stringr before changing here in dbplyr? I also have commented on that issue with another idea (stringr exporting a str_ilike function) 

Perhaps for this pr it can be kept as is, and then changed in the future to reflect changes as and when they happen in stringr? But equally I'm happy to change things here if you prefer?

Reply:
I think it's better to just make the change now — it's technically out of sync but I don't think this function is important enough to manage the updates across multiple package updates.

Reply:
Sure, I have changed the default to FALSE in my latest commit 

(I also opened a pr on stringr to introduce str_ilike https://github.com/tidyverse/stringr/pull/544, but I imagine it would be best to wait for a future version of stringr to be released with that, if you´re happy with it, before adding it in here in dbplyr)